### ISSUE TITLE
Multiple non anonymous operations raise an error

### ISSUE THREAD

**Issue by thecylax** [dailymotion/tartiflette-aiohttp#14]
- **tartiflette**: 0.6.7   
- **tartiflette-aiohttp**: 0.4.2
- **Python version**: 3.7.2
- **Executed in docker**: No

> If a Document contains only one operation, that operation may be unnamed or represented in the shorthand form, which omits both the query keyword and operation name. Otherwise, if a GraphQL Document contains multiple operations, each operation must be named. When submitting a Document with multiple operations to a GraphQL service, the name of the desired operation to be executed must also be provided.

https://graphql.github.io/graphql-spec/June2018/

Request: 
```
query allUsers {
  users {
    name
  }
}

query singleUser {
  user(id: 1) {
    name
  }
}

```

Response:
```
{
  "data": null,
  "errors": [
    {
      "message": "Must provide operation name if query contains multiple operations.",
      "path": null,
      "locations": []
    }
  ]
}
```

**Comment by abusi** [comment]
https://github.com/dailymotion/tartiflette-aiohttp/blob/0b5e8d75fe8f28858a02d4623432bb068831f7dc/tartiflette_aiohttp/_handler.py#L37  The `query_name` here isn't passed to the executor.
I'll fix this soon.
Thank you.

**Comment by thecylax** [comment]
I used the graphiql view provided by Django and the Insomnia desktop application.

**Comment by tsunammis** [comment]
@thecylax this is fixed with the version `0.5.1` :-)
